packages/gitgraph-core/src/branch.ts

```diff
@@ -44,6 +45,17 @@ interface BranchOptions<TNode = SVGElement> {
   onGraphUpdate: () => void;
 }
 
+/**
+ * Type of a function which defines branch order in computed branches paths.
+ * Returns:
+ * - negative number if `branchNameA` should be rendered before `branchNameB`
+ * - positive number if `branchNameA` should be rendered after `branchNameB`
+ */
+type BranchesOrderFunction = (
```

Comment:
I'll move this type into the `BranchesOrder` file as these concepts feels definitely related. It's subtly different from `Branch`, which represent only 1 branch where `BranchesOrder` represent the ordering of all branches in the graph.

I liked that you wrote a comprehensive doc for this type!

Reply:
I might rename this `SortBranches` though. The fact it's a function is already embed in the type. I like to make it explicit what this function is about 👌